docs/supported-languages.md

```diff
@@ -306,7 +306,8 @@ Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your code
 
 _*Semgrep Supply Chain scans transitive dependencies for **all supported languages** but does **not** perform reachability analysis on transitive dependencies._ <br />
 _**‡Reachability support level** refers to the level of support for reachability analysis for the language. At the minimum, Semgrep Supply Chain uses **lockfile-only** rules, which check a package's version against versions with known vulnerabilities._ <br />
-_**††**Semgrep Supply Chain supports `requirements.txt` when it is used as a **lockfile**. This means that `requirements.txt` must be set to exact versions (pinned dependencies) and the file must be generated automatically._
+_**††**Semgrep Supply Chain supports `requirements.txt` when it is used as a **lockfile**. This means that `requirements.txt` must be set to exact versions (pinned dependencies) and the file must be generated automatically._ <br />
+_**§** We do not analyze the transitivity of packages for these language/lockfile combinations. All dependencies will be listed with "Unknown" transitivity._
```

Comment:
```suggestion
_**§** Supply Chain does not analyze the transitivity of packages for these language or lockfile combinations. All dependencies are listed as **Unknown** transitivity._
```
It's ok to use we if we're referring to Semgrep, Inc (a person or org). In this case, "we" refers to the product that performs the analysis, so it's (at least) Supply Chain.